packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -61,7 +61,7 @@ const ROW_HEIGHTS = {
   }
 };
 
-const SELECTION_CELL_DEFAULT_WIDTH = 55;
+const SELECTION_CELL_DEFAULT_WIDTH = 39;
```

Comment:
this seems to be a substantial change, where'd the original width come from? does it need to change if we're in large?

Reply:
There is significant extra whitespace that needed to be removed, a 16px end(right) padding. And good catch because the value does not change between medium and large scale. Seems out of scope for the requested change, but something I should fix anyway. I checked spectrum-css and they use 38px (medium) and 48px (large).

And I think the 55px was wrong, I came up with 39px by subtracting 16px, but if you look at the elements it should be 38px. 55px might have included the border, but we probably moved that during implementation.